common.go

```diff
@@ -153,6 +153,12 @@ func runBuild(c *cli.Context, dir, imageName, dockerfile string, noCache bool) e
 		if noCache {
 			args = append(args, "--no-cache")
 		}
+
+		if buildArgs != nil && len(buildArgs) > 0 {
```

Comment:
FWIW the nil check isn't necessary here as an empty slice has len(0)

Reply:
Thanks!
I will fix.